hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/OMMetadataManagerTestUtils.java

```diff
@@ -222,6 +222,44 @@ public static void writeKeyToOm(OMMetadataManager omMetadataManager,
                     .build());
   }
 
+  @SuppressWarnings("checkstyle:parameternumber")
+  /**
+   * Write a directory as key on OM instance.
+   * We don't need to set size.
+   * @throws IOException
+   */
+  public static void writeKeyToOm(OMMetadataManager omMetadataManager,
```

Comment:
If this is creating a directory then maybe it should be called writeDirToOm().  also, don't copy the implementation from the other method.  "DRY"

Reply:
It's the same without setting the size. We could call the above method with `dataSize=0` to avoid copying the implementation.